future_challenges_for_RSEng.tex

```diff
@@ -147,10 +147,14 @@ \section{Introduction}
 and community in the mid-term future, focusing on future challenges.
 While future projections are common in industry~\cite{Gartner2025},
 so far there is a projection lacking for the RSEng space.
-We present here the results of a workshop discussion at the
+We present here the results of a workshop discussion (notes content is archived here \cite{goth2025_WSPad}) at the
 deRSE25 conference in Karlsruhe\footnote{deRSE25, session ``The End of {RSEng}?
 Challenges and Risks for {RSEng}'': \url{https://events.hifis.net/event/1741/contributions/14026/}},
 without claiming to represent the entire deRSE community.
+The material presented here is based on the output of the workshop participants, but extended with the authors' ideas.
+Since deRSE25 was co-located with SE25, the annual conference for the Software Engineering community in Germany, the discussion was also open to software engineers,
+but a precise statistic of the workshop participants background was not taken.
```

Comment:
```suggestion
Since deRSE25 was co-located with SE25, the annual conference for the Software Engineering community in Germany, the discussion was also open to software engineers,
but no list or profile of the participants is available.
```